DeleteButton.js

```diff
@@ -0,0 +1,25 @@
+export class DeleteButton {
+  constructor(userId, userContainer) {
+    this.url = "https://jsonplaceholder.typicode.com/users";
+    this.userId = userId;
+    this.userContainer = userContainer;
+    this.createButton(userContainer);
+  }
+
+  deleteUser = async () => {
+    const deleteUserResponse = await fetch(`${this.url}/${this.userId}`, {
+      method: "DELETE",
+    });
+    if (this.userContainer) {
+      this.userContainer.remove();
+    }
+  };
```

Comment:
`deleteUserResponse ` is unused.

You need to check if the request was successful before removing the user tile